src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -89,7 +97,18 @@ public static void MatrixTimesSource(bool transpose, AlignedArray matrix, Aligne
             }
         }
 
-        public static void MatrixTimesSource(AlignedArray matrix, ReadOnlySpan<int> rgposSrc, AlignedArray sourceValues,
+        /// <summary>
+        /// Multiplies a matrix times a source.
+        /// </summary>
+        /// <param name="matrix">The input matrix.</param>
+        /// <param name="rgposSrc">The source positions.</param>
+        /// <param name="sourceValues">The source values.</param>
+        /// <param name="posMin">The minimum position.</param>
+        /// <param name="iposMin">The minimum position index.</param>
+        /// <param name="iposLimit">The position limit.</param>
+        /// <param name="destination">The destination matrix.</param>
+        /// <param name="stride">The column stride.</param>
+        public static void MatrixTimesSource(AlignedArray matrix, int[] rgposSrc, AlignedArray sourceValues,
```

Comment:
The failure in the 3.0 ci leg is here. you changed the ```ReadOnlySpan<int>``` to ```int[]``` can you please revert it back ?

Reply:
np @jwood803 :)

Reply:
Thanks @anipik I missed that. Proves the value of this new CI leg.